src/KubeOps/Operator/Webhooks/WebhookConfig.cs

```diff
@@ -0,0 +1,10 @@
+using k8s.Models;
+
+namespace KubeOps.Operator.Webhooks
+{
+    public record WebhookConfig(
```

Comment:
since this is public, the docs should be added for the class

Reply:
Internal again. Man, I'm bad at watching that.